internal/config/config.go

```diff
@@ -88,6 +88,8 @@ type Config struct {
 	ReplicationChannel                      string                       `config:"replication_channel" yaml:"replication_channel"`
 	ExternalReplicationChannel              string                       `config:"external_replication_channel" yaml:"external_replication_channel"`
 	ExternalReplicationType                 util.ExternalReplicationType `config:"external_replication_type" yaml:"external_replication_type"`
+	ASync                                	bool                         `config:"a_sync" yaml:"a_sync"`
+	ASyncAllowedLag                       	int64                        `config:"a_sync_allowed_lag" yaml:"a_sync_allowed_lag"`
```

Comment:
What is `a_sync`?

Reply:
New setting which will allow client to set server in async mode. In async mode in case of failover mysync will promote slave to new master even if it has replication delay, but only if delay less then ASyncAllowedLag